package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ably/ui",
-  "version": "14.7.1",
+  "version": "14.7.1-dev.b7e4d12",
```

Comment:
`14.7.2` has been released now, so we'll need to target `14.7.3` here (maybe coordinate with @aralovelace to see what else that release could have, or whether this goes out by itself).

Also we should avoid dev versions in `package.json`

Reply:
I published `14.7.3` already, so this should be part of `14.7.4`

Reply:
Was mainly just a placeholder for the review app, will bump to `14.7.4`